### PR TITLE
Feat(FRO-76):Create a new thread through reply icon

### PR DESCRIPTION
**Pull request by Mavrik-jnr**
# Fixes Issue/Line
https://linear.app/team-bevel/issue/FRO-76/create-a-new-thread-through-reply-icon-right

https://linear.app/team-bevel/issue/FRO-76/create-a-new-thread-through-reply-icon-right

**My PR Closes #issue_number_here** or
**My PR Fixes FRO-76**

# Changes proposed

## What were you told to do?
Create a new thread via reply icon 

## What did you do?

 I added the comment board as a route to when threads are opened so when a reply icon is clicked the url changes to a thread which then renders the comment board 

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

https://user-images.githubusercontent.com/60897319/203926574-38815c97-2b96-4b11-bf18-b821d7e502d3.mp4

### CODE REVIEW

**Comment by Mavrik-jnr** [comment]
Chat api currently down, so the Message pane with reply icon isn't rendered, The demo above shows what happens when the reply is clicked
error

**Comment by Mavrik-jnr** [comment]
@JulianaSau  @ickynavigator Please Review: 

https://user-images.githubusercontent.com/60897319/204022040-6ceb835e-0273-4b09-9939-913196b531d3.mp4

**Comment by Mavrik-jnr** [comment]
@JulianaSau I've now fixed the comment Board header
https://user-images.githubusercontent.com/60897319/204084477-ac0fc268-0dc3-4539-a28f-57fd7701cd9d.mp4

File: packages/root-config/src/assets/plugin.importmap.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
     "@zuri/zuri-plugin-tools": "https://externaltools.zuri.chat/zuri-zuri-plugin-tools.js",
     "@zuri/zuri-plugin-channels": "https://channels.zuri.chat/static/main/dist/zuri-zuri-plugin-channels.js",
     "@zuri/zuri-plugin-contributions": "https://ct.zuri.chat/js/app.js",
-    "@zuri/zuri-plugin-messaging": "https://chat.zuri.chat/zuri-messaging.js"
+    "@zuri/zuri-plugin-messaging":"http://localhost:5013/zuri-messaging.js"
```
Comment (billmal071) [inline packages/root-config/src/assets/plugin.importmap.json:17]: Change it back to the live link.

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ const Sidebar = props => {
         document.querySelector("body").style.cursor = "col-resize";
```
Comment (billmal071) [inline packages/main/src/components/protected/sidebar/components/Sidebar.jsx:51]: Make use of refs if you want to make changes to the dom.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by Ginohmk** [comment]
[Linear Ticket Hydraulic](https://linear.app/team-hydraulics/issue/FRO-55/thread-sidebar)

[Linear Ticket Bevel](https://linear.app/team-bevel/issue/FRO-76/create-a-new-thread-through-reply-icon-right)

@JulianaSau @billmal071 @ickynavigator 
https://user-images.githubusercontent.com/58771507/204100200-6fb13469-404d-4717-acd4-7c7c94384afb.mp4

File: packages/root-config/src/assets/plugin.importmap.json (hunk 1 above)
Comment (Mavrik-jnr) [reply packages/root-config/src/assets/plugin.importmap.json]: hosted link is back @billmal071

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx (hunk 2 above)
Comment (Mavrik-jnr) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx:51]: okay, we didn't make these changes

Comment (Mavrik-jnr) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx:51]: ok @billmal071  we didn't make those changes

**Comment by billmal071** [comment]
> [Linear Ticket Hydraulic](https://linear.app/team-hydraulics/issue/FRO-55/thread-sidebar)
> 
> [Linear Ticket Bevel](https://linear.app/team-bevel/issue/FRO-76/create-a-new-thread-through-reply-icon-right)
> 
> @JulianaSau @billmal071 @ickynavigator
> 
>  video2625537709.mp4

I can see an overflow on the comments, can you look into it and fix it.

**Comment by Mavrik-jnr** [comment]
@billmal071 @JulianaSau I've fixed the comment overflow

https://user-images.githubusercontent.com/60897319/204121738-f6ac06f6-5075-4a40-9a5b-f25040888ef6.mp4

**Comment by JulianaSau** [comment]
Hi @Mavrik-jnr @Ginohmk, great work. The message board on the comment board should be fixed to the bottom, the same way it to on the message board to the left

**Comment by Mavrik-jnr** [comment]
@JulianaSau @billmal071 Message Pane Fixed
https://user-images.githubusercontent.com/60897319/204127381-fe06c47a-5be6-4c4f-956b-917009d6ea0b.mp4

**Review by JulianaSau — APPROVED** [review]

File: packages/ui/src/comment-board/CommentBoard.styled.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ const CommentBoardWrapper = styled.aside`
   top: -8.5px;
   height: 93vh;
   padding: 0 0 20px 0;
+  padding-right: 1.5rem;
```
Comment (billmal071) [inline packages/ui/src/comment-board/CommentBoard.styled.js]: Why this use just padding and set what you want.

File: packages/ui/src/comment-board/CommentBoard.jsx [hunk 4]
```diff
@@ -88,4 +147,24 @@ const CommentBoard = ({ commentBoardConfig, Messages = [] }) => {
...
+
+//     <p className="time_ago">6 hours ago</p>
+//   </div>
+//   <CommentMessageItem>
+//     <RichTextRenderer
+//       richUiMessageConfig={message.richUiData}
+//     />
+//   </CommentMessageItem>
+// </div>
+// </div>
```
Comment (billmal071) [inline packages/ui/src/comment-board/CommentBoard.jsx]: Remove.

**Comment by billmal071** [comment]
Hi @Mavrik-jnr @Ginohmk you should fix the new issues I pointed out in a new pr, then also ensure the messages can't seen from under the message pane, that's for threads. I can see that when you scroll up, the messages are already showing under the message input. It should just just after it clears the message input.

File: packages/ui/src/comment-board/CommentBoard.jsx (hunk 4 above)
Comment (Mavrik-jnr) [reply packages/ui/src/comment-board/CommentBoard.jsx]: Okay, my last commit cleaned this up

File: packages/ui/src/comment-board/CommentBoard.styled.js (hunk 3 above)
Comment (Mavrik-jnr) [reply packages/ui/src/comment-board/CommentBoard.styled.js]: Okay

**Pull request merged**